storage/sqlite/sqlite.js

```diff
@@ -61,6 +61,23 @@ module.exports = function(RED) {
                         }
                     }
                 }
+                if (this.sqlquery == "batch") {
+                    if (typeof msg.topic === 'string') {
+                        node.mydbConfig.db.exec(msg.topic, function(err) {
+                            if (err) { node.error(err,msg);}
+                            else {
+                                delete msg.payload;
```

Comment:
I think it should return an empty array on success (like other modes). So at least it contains something...